src/controllers/comments.controller.js

```diff
@@ -35,6 +38,7 @@ class Comments {
         parentCommentId: req.body.parentCommentId,
       };
       const createdComment = await commentsService.addComment(comment);
+      notifyUsersWhoFavorited(req, res, getArticle.id, req.params.slug);
```

Comment:
This function is sending an email as well. You can add await keyword in order to be consistent

Reply:
Fixed, please check again